hbase-balancer/src/test/java/org/apache/hadoop/hbase/master/balancer/StochasticBalancerTestBase.java

```diff
@@ -70,38 +69,10 @@ protected void testWithClusterWithIteration(int numNodes, int numRegions, int nu
       assertFullyBalancedForReplicas);
   }
 
-  protected void testWithCluster(Map<ServerName, List<RegionInfo>> serverMap,
```

Comment:
I don't think there are any tests where it actually makes sense to run without iteration